src/base/os.lua

```diff
@@ -834,6 +834,7 @@
 		["linux"]      = { "linux",   "posix", "desktop" },
 		["macosx"]     = { "macosx",  "darwin", "posix", "desktop" },
 		["solaris"]    = { "solaris", "posix", "desktop" },
+		["tvos"]       = { "tvos",    "darwin", "posix", "mobile" },
```

Comment:
This is something I'm not quite sure on, what tags should tvOS have?

An Apple TV isn't a desktop device and it isn't a mobile device, but an Apple TV's internals are very similar to an iPhone, and iOS and tvOS share a lot of common history and functionality. So because of the familiarity to iOS I tagged it as `mobile` but I'm open to other suggestions